src/features/pass-survey/ui/ActivityItem.tsx

```diff
@@ -400,6 +448,16 @@ function ActivityItem({
           }
         }}
       >
+        {assignment && assignment.respondent.id !== assignment.target.id && (
+          <XStack mx={16} mb={12}>
+            <ActivityAssignmentBadge
+              assignment={assignment}
+              accessibilityLabel="item_display_assignment"
+            />
+            <Box flexGrow={1} flexShrink={1} />
+          </XStack>
+        )}
```

Comment:
The multi-informant badge is meant to be inserted into the item's markdown, below any paragraphs that just contain media (images, etc.) and directly above the first paragraph that contains text. Check out the web app to see how this was done; hopefully you can just reuse the same utility function [`insertAfterMedia`](https://github.com/ChildMindInstitute/mindlogger-web-refactor/pull/534/files#diff-b71cf5aa4da256f58f6fca6f8557c99cba908811493da204113a2a90445d1111) from the web app to achieve this! You can even copy the unit tests over.

Reply:
It seems to work pretty well 💁‍♀️ 

![iPhone 15 Pro Max 2024-10-06 12-02-12](https://github.com/user-attachments/assets/fb29aa39-5e63-4d73-8599-0395893f0259)